Application/EdFi.Admin.DataAccess/Utils/DefaultApplicationCreator.cs

```diff
@@ -4,8 +4,7 @@
 // See the LICENSE and NOTICES files in the project root for more information.
 
 using System.Collections.Generic;
-using System.Data.Entity;
-using System.Data.Entity.Migrations;
+using Microsoft.EntityFrameworkCore;
```

Comment:
Reorder using statements to get System.* on top and the rest of the libraries in alphabetic order.